src/platform/utilities/tests/sso.unit.spec.js

```diff
@@ -117,6 +117,7 @@ describe('checkAutoSession', () => {
     sandbox.stub(profUtils, 'hasSession').returns(false);
     sandbox.stub(profUtils, 'hasSessionSSO').returns(true);
     sandbox.stub(forceAuth, 'getForceAuth').returns(undefined);
+    mockFetch();
```

Comment:
With the change to `resetFetch()` we need to make sure that `mockFetch()` is called before working with the `global.fetch` stub.

Reply:
@hinzed1127 @ericbuckley I had to make this tiny change to your test file